src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $white;
+
+  .card__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+  }
+
+  .card__title {
+    margin: 0;
+    margin-bottom: 4px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-style: normal;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-text-color;
+  }
+
+  .card__id {
+    margin: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-family: Roboto, sans-serif;
```

Comment:
it seems like you're repeating this line a bunch of time, maybe set it as global font?